### ISSUE TITLE
Performance

### ISSUE THREAD

**Issue by simonnuss** [cwebbbi/PowerBITextAnalytics#4]
I am attempting to run the extension over ~30K distinct sentences.  It's been running for about 2 hours now.  Has it been testing on large volumes of data?

**Comment by cwebbbi** [comment]
Yes, but not since the changes that were made to the API late last year. Are you using the free tier? There were some major changes made to the licensing that are described here: https://datachant.com/2017/11/17/wrong-microsoft-text-analytics/
If you are hitting the limit on the free tier then the connector might not be reporting errors properly.
